locale/ar_IQ/admin.xml

```diff
@@ -0,0 +1,133 @@
+	<message key="admin.settings.introduction">المقدمة</message>
+	<message key="admin.settings.minPasswordLength">أقل طول لكلمة المرور</message>
+	<message key="admin.settings.oaiRegistration">سجِّل الموقع لأغراض الفهرسة (حصاد البيانات الوصفية)</message>
+	<message key="admin.settings.siteLanguage">لغة الموقع</message>
+	<message key="admin.settings.siteStyleSheetInvalid">صيغة ملف أنماط الموقع غير صحيحة. الصيغة المقبولة هي .css.</message>
+	<message key="admin.settings.siteStyleSheet">ملف أنماط الموقع</message>
+	<message key="admin.settings.siteLogo">شعار الموقع</message>
+	<message key="admin.settings.siteTitle">عنوان الموقع</message>
+	<message key="admin.settings.siteTheme">نمط الموقع</message>
+	<message key="admin.siteAdmin">إدارة الموقع</message>
+	<message key="admin.siteManagement">إدارة الموقع</message>
+	<message key="admin.siteSettings">إعدادات الموقع</message>
+	<message key="admin.siteSetup">تهيئة الموقع</message>
+	<message key="admin.systemConfigFileReadError"><![CDATA[إن ملف الإعدادات <tt>config.inc.php</tt> غير موجود، أو تتعذر قراءته، أو غير صحيح.]]></message>
+	<message key="admin.systemInformation">معلومات النظام</message>
+	<message key="admin.versionBuild">بناء</message>
+	<message key="admin.version.checkForUpdates">التحقق من وجود تحديثات</message>
+	<message key="admin.version.downloadPackage">التنزيل</message>
+	<message key="admin.version.downloadPatch">حزمة التنزيل</message>
+	<message key="admin.versionHistory">تأريخ النسخ</message>
+	<message key="admin.version.latest">النسخة الأخيرة</message>
+	<message key="admin.versionMajor">الرئيسي</message>
+	<message key="admin.versionMinor">الفرعي</message>
+	<message key="admin.version.moreInfo">مزيد من المعلومات</message>
+	<message key="admin.versionRevision">المراجعة</message>
+	<message key="admin.version.updateAvailable">تتوفر نسخة محدثة</message>
+	<message key="admin.version.upToDate">إن نظامك مواكب لأحدث نسخة</message>
+	<message key="admin.version">النسخة</message>
+	
+	<message key="admin.fileLoader.wrongBasePathLocation">المسار الأساسي {$path} ينبغي أن يكون ضمن حافظة الملفات العمومية.</message>
+	<message key="admin.fileLoader.pathNotAccessible">حافظة الملفات {$path} ليست مساراً صحيحاً أو تتعذر قراءتها.</message>
+	<message key="admin.fileLoader.moveFileFailed">الملف {$filename} تعذر نقله من {$currentFilePath} إلى {$destinationPath}</message>
+	<message key="admin.fileLoader.fileProcessed">الملف {$filename} قد تمت معالجته وحفظه.</message>
+	<message key="admin.fileLoader.emailSubject">محمِّل الملفات</message>
+	<message key="admin.copyAccessLogFileTool.usage">
+		إنسخ ملفات سجل التوثيق السابقة للملقم apache مع قيامك بتصفية الإدخالات المتعلقة بهذا التنصيب،
+		وضعها في حافظة الملفات الحالية ضمن موضع الإضافة UsageStatsPlugin. هذا يجب أن يقوم به مستخدم يمكل صلاحيات كافية تسمح له بقراءة هذا النوع من الملفات.
+
+		هذه الأداة ستقوم بمجرد نسخ الملفات التي لم يسبق لها أن تكون ضمن حوافظة ملفات الإضافة usageStats وهي: (stage, processing, archive, reject).
```

Comment:
These are folder names, aren't they?
I kept the English words because the system will be using these names regardless of the interface language.